website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,131 @@
+---
+layout: docs
+page_title: Consul API Gateway Basic Usage
+description:  >-
+  Consul API Gateway Basic Usage
+---
+
+
+# Basic Usage
+
+1. Verify that the [requirements](/docs/api-gateway/consul-api-gateway-install#requirments) have been met.
+1. Verify that the Consul API Gateway CRDs and controller have been installed and applied (see [Installation](/docs/api-gateway/consul-api-gateway-install)).
+1. Configure the artifacts described below in [Configuration](/docs/api-gateway/configuration).
+
+<CodeBlockConfig hideClipboard filename="values.yaml">
+
+  ```yaml
+  apiGateway:
+  managedGatewayClass:
+  enabled: true
+  ```
+
+</CodeBlockConfig>
+
+1. Issue the `kubectl apply` command to implement the configurations, e.g.:
+
+```shell-session
+$ kubectl apply -f gateway.yaml routes.yaml
+   ```
```

Comment:
Indentation here is inconsistent, but I'm not 100% sure which level is what we want